contracts/StandardBounties.sol

```diff
@@ -509,13 +509,12 @@ contract StandardBounties {
     public
     senderIsValid(_sender)
     validateBountyArrayIndex(_bountyId)
-    validateIssuerArrayIndex(_bountyId, _issuerId)
```

Comment:
Why did you remove this line? This validates the array index of the sender of the message, which is different from the index of the issuer who needs to be changed.

Reply:
Stack too deep issue started to happen once I made the changes of the emitted values. I thought this one can be compromised since we are checking for it indirectly using onlyIssuer()

Reply:
We should figure out why this happened, because `validateIssuerArrayIndex` is used in many places in the contract.

Reply:
It's because https://ethereum.stackexchange.com/questions/6061/error-while-compiling-stack-too-deep